hm_pyhelper/diagnostics/diagnostics_report.py

```diff
@@ -16,6 +16,16 @@ class DiagnosticsReport(dict):
         'public_key', 'bluetooth', 'VARIANT', 'FREQ', 'serial_number'
     }
 
+    KEYS_TO_CHECK_IN_GATEWAY_TRANSACTION_PAYLOAD = {
+        'address', 'fee', 'owner', 'payer', 'txn', 'staking fee'
+    }
+
+    ADD_GATEWAY_TXN_NAME_KEY = 'destination_name'
```

Comment:
This is mixing the naming prefixes `add_gateway_` and `destination_`. The `destination_name` does not really have anything to do with the transaction. We could have decided to start off with pallet tracking without implementing any transaction logic.

I'd rename this `DESTINATION_NAME_KEY` and the other to `DESTINATION_ADD_GATEWAY_TXN_KEY`. (I have some more nuanced thoughts if you want to jump on a call).

Reply:
Moved out of hm-pyhelper now. I'll update these names at HPT end.